app/frontend/src/features/profile/view/Overview.tsx

```diff
@@ -129,17 +135,21 @@ export default function Overview({ user, setIsRequesting }: OverviewProps) {
           ]
         }
       />
-      <Divider />
-      <BarWithHelp
-        value={user.communityStanding || 0}
-        label={COMMUNITY_STANDING}
-        description={COMMUNITY_STANDING_DESCRIPTION}
-      />
-      <BarWithHelp
-        value={user.verification || 0}
-        label={VERIFICATION_SCORE}
-        description={VERIFICATION_SCORE_DESCRIPTION}
-      />
+      <Divider margBottom />
+      <div className={classes.bar}>
+        <BarWithHelp
```

Comment:
If this component doesn't support `className` yet, better add it in rather than wrap it with a div